subsystem/host_test.go

```diff
@@ -690,19 +690,23 @@ var _ = Describe("Host tests", func() {
 		Expect(err).NotTo(HaveOccurred())
 		h := getHost(*cluster2.GetPayload().ID, *hostID)
 
-		// register again to cluster 2 and expect it to be in discovery status
-		Expect(db.Model(h).Update("status", "known").Error).NotTo(HaveOccurred())
-		h = getHost(*cluster2.GetPayload().ID, *hostID)
-		Expect(swag.StringValue(h.Status)).Should(Equal("known"))
+		Eventually(func() string {
+			h = getHost(*cluster2.GetPayload().ID, *hostID)
+			return swag.StringValue(h.Status)
+		}, "30s", "1s").Should(Equal(models.HostStatusKnown))
```

Comment:
how will the host be known if inventory is empty? you need to call all the step reply in order to make it known